psydac/api/tests/test_api_2d_scalar_multi_patch.py

```diff
@@ -111,8 +111,8 @@ def test_poisson_2d_2_patch_dirichlet_1():
 
     l2_error, h1_error = run_poisson_2d(solution, f, domain, ncells=[2**2,2**2], degree=[2,2])
 
-    expected_l2_error = 0.0010176148331990707
-    expected_h1_error = 0.028398193995821736
+    expected_l2_error = 0.002035229666394183
+    expected_h1_error = 0.056796387991647795
```

Comment:
Why have you multiplied the errors by a factor of two? Was the original code not summing up the contributions from the two patches?

Reply:
yes, that was a bug in the code, it was not summing up the contributions